code/datums/abilities/flock/flockmind.dm

```diff
@@ -167,12 +167,42 @@
 
 	logTheThing(LOG_COMBAT, holder.get_controlling_mob(), "designates [constructTarget(M)] as [flock.isEnemy(M) ? "" : "not "]an enemy at [log_loc(src.holder.owner)].")
 
-	if (flock.isEnemy(M))
+	if (flock.isIgnored(M))
+		flock.removeIgnore(M)
+	else if (flock.isEnemy(M))
 		flock.removeEnemy(M)
 		return
 
 	flock.updateEnemy(M)
 
+
+/datum/targetable/flockmindAbility/designateIgnore
+	name = "Designate ignore"
+	desc = "Designate someone to be ignored by your Flock."
+	icon_state = "designate_ignore"
+	cooldown = 0.1 SECONDS
+
+/datum/targetable/flockmindAbility/designateIgnore/cast(atom/target)
+	if(..())
+		return TRUE
+
+	var/mob/living/intangible/flock/F = holder.owner
+
+	if (!isflockvalidenemy(target))
+		boutput(F, "<span class='alert'>That isn't a valid target.</span>")
+		return TRUE
+
+	if (!F.flock)
+		return TRUE
+
+	logTheThing(LOG_COMBAT, holder.owner, "designates [constructTarget(target)] to [F.flock.isIgnored(target) ? "" : "not "] be ignored by their flock at [log_loc(src.holder.owner)].")
+
+	if (F.flock.isIgnored(target))
+		F.flock.removeIgnore(target)
+	else
+		F.flock.removeEnemy(target)
+		F.flock.addIgnore(target)
+
```

Comment:
```suggestion
	if (F.flock.isIgnored(target))
		F.flock.removeIgnore(target)
		return
	if (F.flock.isEnemy(target))
		F.flock.removeEnemy(target)
	
	F.flock.addIgnore(target)

```
Since no enemy check on removeEnemy() right now

Reply:
But why does that matter? It's not going to runtime and the overhead from checking is almost certainly worse than just calling it.

Reply:
Checking A in enemies seemed like less stuff than checking through annotations